docker/bootstrap/CHANGELOG.md

```diff
@@ -162,3 +162,4 @@ List of changes between bootstrap image versions.
 ## [41] - 2025-01-15
 ### Changes
 - Update base image to bookworm
+- Add MySQL84 image
```

Comment:
@frouioui Do we need to bump this to 42 now that we add this new image?

Reply:
For me we don't as we add a new image and not editing a new one, but it's only my opinion.